.github/workflows/build_and_test.yml

```diff
@@ -42,3 +42,32 @@ jobs:
             --packages-up-to obstacle_stop_planner_refine \
             --event-handlers console_cohesion+ \
             --return-code-on-test-failure
+
+  code-metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Measure code metrics
+        run: |
+          git clone https://github.com/terryyin/lizard.git
+          mkdir public
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
```

Comment:
I'd just like to recommend to add `-l python`, which might be needed for other packages.

Reply:
added e5753b0 :+1: